components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -181,6 +214,7 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem animated:(BOOL)animated {
     [self updateTitleColorForAllViews];
     [self updateImageTintColorForAllViews];
     [self updateTitleFontForAllViews];
+    [self didSelectItemAtIndex:NSNotFound animateTransition:animated];
```

Comment:
Nit: What does selecting an NSNotFound item do?

Reply:
It makes nothing selected.